packages/pagination/pagination.stories.tsx

```diff
@@ -39,7 +39,7 @@ export const ExampleWPagedListControlledComponent = () => {
         <>
           <BorderedList tag="ul">
             {initialData.slice(itemStartIndex, itemEndIndex).map(name => (
-              <li>{name}</li>
+              <li key={name}>{name}</li>
```

Comment:
I think adding some of these keys will address some console warnings/errors we're seeing in Kommander too 👍